webpack.common.js

```diff
@@ -20,20 +20,29 @@ module.exports = {
 				test: /\.scss$/,
 				use: ['vue-style-loader', 'css-loader', 'sass-loader']
 			},
-			{
-				test: /\.(js|vue)$/,
-				use: 'eslint-loader',
-				exclude: /node_modules/,
-				enforce: 'pre'
-			},
+			// {
+			// 	test: /\.(js|vue)$/,
+			// 	use: 'eslint-loader',
+			// 	exclude: /node_modules/,
+			// 	enforce: 'pre'
+			// },
 			{
 				test: /\.vue$/,
 				loader: 'vue-loader',
 				exclude: /node_modules/
 			},
 			{
 				test: /\.js$/,
-				loader: 'babel-loader',
+				use: {
+					loader: 'babel-loader',
+					options: {
+						plugins: [
+							'@babel/plugin-syntax-dynamic-import',
+							'@babel/plugin-proposal-object-rest-spread'
+						],
+						presets: ['@babel/preset-env']
```

Comment:
I don't think this is necessary since be have the shared browserslist-config. Correct me if I'm wrong though.

Reply:
Not sure, will look into it